src/containers/StudyCreateForm/StudyCreateForm.tsx

```diff
@@ -178,166 +150,296 @@ function StudyCreateFormContainer() {
         setThumbnailImage(result);
 
         resolve();
+        setIsLoading(false);
       };
     });
   };
 
   const onImageChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    if (e.target.files && e.target.files[0]) {
-      const maxSize = 1 * 1024 * 1024;
-      const fileSize = e.target.files[0].size;
+    const { files } = e.target;
+
+    if (!files || !files[0]) return;
+
+    const maxSize = 1 * 1024 * 1024;
+    const fileSize = files[0].size;
 
-      if (fileSize > maxSize) {
-        alert('첨부 파일은 최대 1MB 입니다.');
-        return;
-      }
-      setImageSrc(e.target.files[0]);
-      encodeFile(e.target.files[0]);
+    if (fileSize > maxSize) {
+      setFileErrorMessage('파일 크기는 최대 1MB 입니다.');
+      return;
     }
+
+    setImageSrc(files[0]);
+    encodeFile(files[0]);
+    setFileErrorMessage('');
   };
 
   const isOnlineChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    if (e.target.value === 'online') {
-      setIsOnline(true);
-      setLocation('');
-    } else {
-      setIsOnline(false);
-    }
+    const { value } = e.target;
+    setIsOnline(value === 'online' ? true : false);
   };
 
-  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
-    console.log('제목 ' + title);
-    console.log('카테고리 ' + category);
-    console.log('지역 ' + isOnline + ' ' + location);
-    console.log('인원 ' + peopleNumber);
-    console.log('스터디 기간 ' + startDate + '~' + endDate);
-    console.log('선호 MBTI ' + mbtiCheckedList);
-    console.log('이미지 ' + imageSrc);
-    console.log('내용' + body);
+  useEffect(() => {
+    if (topic) setTopicErrorMessage('');
+  }, [topic]);
+
+  useEffect(() => {
+    if (recruitsNumber) setPeopleErrorMessage('');
+  }, [recruitsNumber]);
```

Comment:
p4;
2개로 나눈 이유가 궁금합니다. :) 
저도 React가 약해서 ㅜㅜ

```suggestion
  useEffect(() => {
    if (topic) setTopicErrorMessage('');
    if (recruitsNumber) setPeopleErrorMessage('');
  }, [topic, recruitsNumber]);
```

Reply:
의존성 배열에 들어가는 내용이 달라서 그러신 것 같습니다! 😁

팽님이 서제스트 해주신 내용대로라면, `recruitsNumber, topic` 중 하나의 값만 바뀌어도 두 에러메시지가 모두 초기화됩니다.

Reply:
맥키님이 말씀해주신 대로 의도했습니다! `topic`과 `recruitsNumber` 에러 메세지를 초기화 하려고 분리했는데 더 나은 방법은 생각해내지 못해서..😓

Reply:
아하 감사합니다. :)